src/libertem/corrections/detector.py

```diff
@@ -121,10 +122,12 @@ def flatten_filter(excluded_pixels, repairs, sig_shape):
     "good" pixels.
     '''
     excluded_flat = np.ravel_multi_index(excluded_pixels, sig_shape)
-    repair_flat = tuple([
+    repair_flat = List()
```

Comment:
Need to also fix the empty case to have consistent types (otherwise we compile twice) - I'm not sure how/if it is possible to assign a type to an empty `List`